alpha/apps/kaltura/lib/kObjectDeleteHandler.php

```diff
@@ -168,6 +177,17 @@ protected function kuserDelete(kuser $kuser)
 				kJobsManager::addDeleteJob($kuser->getPartnerId(), DeleteObjectType::GROUP_USER, $filter);
 			}
 		}
+		
+		$ueFilter = new UserEntryFilter();
+		$ueFilter->set("_eq_user_id", $kuser->getPuserId());
+		
+		$c = new Criteria();
+		$c->add(UserEntryPeer::KUSER_ID, $kuser->getId());
+		if(!UserEntryPeer::doSelectOne($c)) {
+			return;
```

Comment:
we break out of the function entirely if there's not UserEntry record in the DB. The code after the if-statement will not run.